docs/components/custom_component.md

```diff
@@ -112,11 +114,36 @@ ENTRYPOINT ["fondant", "execute", "main"]
 
 ## Requirements.txt
 
-A `requirements.txt` file lists the Python dependencies of the component. Note that any Fondant component will always have `Fondant` as the minimum requirement. It's important to also pin the version of each dependency to make sure the component remains working as expected. Below is an example of a component that relies on several Python libraries such as Pillow, PyTorch and Transformers.
+A `requirements.txt` file lists the Python dependencies of the component. Note that any Fondant
+component will always have `Fondant` as the minimum requirement. It's important to also pin the
+version of each dependency to make sure the component remains working as expected. Below is an
+example of a component that relies on several Python libraries such as Pillow, PyTorch and
+Transformers.
 
 ```
 fondant
 Pillow==10.0.1
 torch==2.0.1
 transformers==4.29.2
 ```
+
+## Build the component
```

Comment:
Maybe add that the tag in the  referenced component yaml will be updated as well so that the next pipeline run will refer to the right image